src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -1349,7 +1349,7 @@ public void canCheckInLostAndPaidItem() {
     assertThat(itemsClient.getById(item.getId()).getJson(), isAvailable());
 
     waitAtMost(1, SECONDS)
-      .until(FakePubSub::getPublishedEvents, hasSize(6));
+      .until(FakePubSub::getPublishedEvents, hasSize(5));
```

Comment:
why?

Reply:
Because this test involves declaring item as lost, which closes the loan without charging lost item fees. This used to trigger publishing of two events: ITEM_DECLARED_LOST and LOG_RECORD (see `EventPublisher#publishDeclaredLostEvent`), but now in this case only one `LOAN_CLOSED` is published instead